packages/data-point/lib/entity-types/entity-request/resolve.test.js

```diff
@@ -324,7 +324,7 @@ describe('resolve', () => {
     })
   })
 
-  test('inject locals value with string template', () => {
+  test('inject locals value with string template | contains object with {url : }', () => {
```

Comment:
This test uses locals to create a dynamic `url` string for the request. We don't need the stuff after the pipe.